manifests/d/Donkz/RemoteDesktopPlus/7.0.1/Donkz.RemoteDesktopPlus.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Donkz.RemoteDesktopPlus
+PackageVersion: 7.0.1
+PackageLocale: en-US
+Publisher: Ruud Hanegraaf

```

```suggestion
Publisher: Donkz (Ruud Hanegraaf)
```
This must match Control Panel.